### PR TITLE
add fuzz reader

### PR DESCRIPTION
**Pull request by adamdecaf**

### CODE REVIEW

File: test/fuzz-reader/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+Otherwise, run `make` and watch the output. Fix any crashes that happen, thanks!
+
+See the `go-fuzz` project for more docs: https://github.com/dvyukov/go-fuzz
+
+### Corpus
+
+Right now our corpus exists mostly of test files. As a machine runs go-fuzz files are written to the `corpus/` directory.
+
+To load all test files we read run: `ln -s ../../../test/ach-*-read/*.ach .` from the `corpus/` directory.
```
Comment (adamdecaf) [inline test/fuzz-reader/README.md:19]: Can we fix the mentions of `ach` here?

See: https://github.com/moov-io/iso8583/commit/047b947304d2d6e75c99b9f3216b505360065e29#r44674961

Comment (mfdeveloper508) [reply test/fuzz-reader/README.md]: fixed

File: test/fuzz-reader/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+### Corpus
+
+Right now our corpus exists mostly of test files. As a machine runs go-fuzz files are written to the `corpus/` directory.
+
+To load all test files we read run: `ln -s ../../../test/testdata/*.dat .` from the `corpus/` directory.
+
+### Downloading crashers from Kubernetes cluster
+
+If the [`moov/achfuzz`](https://hub.docker.com/r/moov/achfuzz) Docker image is running in a Kubernetes cluster you can download the crashers from a mounted volume by executing the following commands.
```
Comment (adamdecaf) [inline test/fuzz-reader/README.md:23]: can we change this naming to `iso8583fuzz` ?

**Comment by adamdecaf** [comment]
@mfdeveloper508 Looks good! Can we include the `Dockerfile-fuzz` file so we can build the docker image? There's a task in the ach makefile to build this as well.

https://github.com/moov-io/ach/blob/master/makefile#L55-L57
https://github.com/moov-io/ach/blob/master/Dockerfile-fuzz

File: test/fuzz-reader/README.md (hunk 2 above)
Comment (mfdeveloper508) [reply test/fuzz-reader/README.md]: sure, changed

**Comment by adamdecaf** [comment]
Approved, thanks!

**Pull request merged**